openhands/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -14,19 +14,14 @@ ENV POETRY_VIRTUALENVS_PATH=/openhands/poetry \
 
 # Install base system dependencies
 RUN apt-get update && \
-    apt-get upgrade -y && \
```

Comment:
The base image has an upgrade like this and is run fairly regularly so this should be redundant. It made a difference in findings before but maybe less of an issue with slim.

https://github.com/nikolaik/docker-python-nodejs/blob/main/templates/debian.Dockerfile#L12